Modules loaded
Done copying.
Unzipping..
Python 3.8.1
Python code starts now!
dataset : /scratch/Books_5.json
pkl_path : /home/hbartsch/thesis-user-modelling/datasets/out-pickle
batch_size : 128
max_seq_length : 512
drop_org_reviews : 0
Device: cuda:0
30522
Start reading: /scratch/Books_5.json
Traceback (most recent call last):
  File "read_json.py", line 304, in <module>
    main(config)
  File "read_json.py", line 285, in main
    user_dict, item_dict = preprocessDF_lisa(data_path, pkl_path, bert_tokenizer, bert_model,
  File "read_json.py", line 218, in preprocessDF_lisa
    df = pd.read_json(path, orient='index')
  File "/home/hbartsch/.conda/envs/thesis-user-modelling/lib/python3.8/site-packages/pandas/io/json/_json.py", line 592, in read_json
    result = json_reader.read()
  File "/home/hbartsch/.conda/envs/thesis-user-modelling/lib/python3.8/site-packages/pandas/io/json/_json.py", line 717, in read
    obj = self._get_object_parser(self.data)
  File "/home/hbartsch/.conda/envs/thesis-user-modelling/lib/python3.8/site-packages/pandas/io/json/_json.py", line 739, in _get_object_parser
    obj = FrameParser(json, **kwargs).parse()
  File "/home/hbartsch/.conda/envs/thesis-user-modelling/lib/python3.8/site-packages/pandas/io/json/_json.py", line 849, in parse
    self._parse_no_numpy()
  File "/home/hbartsch/.conda/envs/thesis-user-modelling/lib/python3.8/site-packages/pandas/io/json/_json.py", line 1105, in _parse_no_numpy
    loads(json, precise_float=self.precise_float),
ValueError: Trailing data
Done!
